.circleci/config.yml

```diff
@@ -1,55 +1,255 @@
+      command: |
+        bash "${HOME}/catkin_ws/src/fixposition_driver/configurations/install_greenzie_rosdep_source.sh"
+        rosdep update
+        rosdep install -iy --from-paths /root/catkin_ws/src --rosdistro=${ROS_DISTRO}
+
+arm64_source_greenzie_setup: &arm64_source_greenzie_setup
+  - run:
+      name: "Install Greenzie ROSDEP Dependencies"
+      command: |
+        mv "${HOME}/catkin_ws/src/fixposition_driver/configurations/10-greenzie-base-arm64.yaml" "${HOME}/catkin_ws/src/fixposition_driver/configurations/10-greenzie-base.yaml"
+        bash "${HOME}/catkin_ws/src/fixposition_driver/configurations/install_greenzie_rosdep_source.sh"
+        rosdep update
+        rosdep install -iy --from-paths /root/catkin_ws/src --rosdistro=${ROS_DISTRO}
+
+
+########################
+######### STEPS ########
+########################
+ros_env_prep_steps: &ros_env_prep_steps
+  # This 'when' is hack which allows us to concatenate list of steps
+  # We cannot combine lists, so this is the best solution I have
+  # See: https://circleci.com/docs/introduction-to-yaml-configurations/#anchors-and-aliases
+  when:
+    condition:
+      equal: [ "true", "true" ]
+    steps:
+      - when:
+          condition:
+            not:
+              matches:
+                pattern: "^(noetic|release-candidate-v.*)$"
+                value: << pipeline.git.branch >>
+          steps:
+            - add_package_server:
+                server_type: "experimental"
+      - when:
+          condition:
+            matches:
+              pattern: "^(release-candidate-v.*)$"
+              value: << pipeline.git.branch >>
+          steps:
+            - add_package_server:
+                server_type: "rc"
+      - <<: *update_environment
+      - <<: *preparing_ros_workspace
+      - <<: *checkout_codebase
+      - <<: *source_ros_setup
+      # arm64 needs some modifications before setting up greenzie
+      # check arm64_source_greenzie_setup for more details
+      - unless:
+          condition:
+            matches:
+              pattern: "^(arm64)$"
+              value: << parameters.architecture >>
+        steps:
+          - <<: *source_greenzie_setup
+      - when:
+          condition:
+            matches:
+              pattern: "^(arm64)$"
+              value: << parameters.architecture >>
+          steps:
+            - <<: *arm64_source_greenzie_setup
+
+debian_build_steps: &debian_build_steps
+  # This 'when' is hack which allows us to concatenate list of steps
+  # We cannot combine lists, so this is the best solution I have
+  # See: https://circleci.com/docs/introduction-to-yaml-configurations/#anchors-and-aliases
+  when:
+    condition:
+      equal: [ "true", "true" ]
     steps:
-      - checkout
       - run:
-          name: "Generate Changelogs"
-          command: greenzie-release changelog -r "${ROS_DISTRO}"
+          name: Generate Changelogs
+          command: |
+            cd ./src/robotic-lawnmower
```

Comment:
Something seems off about going into the `robotic-lawnmower` directory here. That repo shouldn't be a dependency of fixposition at all, particularly since this fork is public and that's a private repo.